<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="buttonOne">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>One</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonTwo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Two</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFour">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Four</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFive">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Five</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSeven">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Seven</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNine">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Nine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonThree">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Three</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
